Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,1897 @@ label mas_wrs_twitch:
+    return       
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_patreon",
+            category=['patreon'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_patreon:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Anyone in particular you're supporting?",
+           "Helping to support someone? You're so kind, [player]!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_patreon')
+    return       
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_urbandictionary",
+            category=['urbandictionary'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_urbandictionary:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Ah, this website is...definitely something.",
+           "You might want to go to a more...trusted source\nif you're looking for accurate definitions, [player]."
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_urbandictionary')
+    return                   
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_python",
+            category=['python.org'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_python:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Are you intrested in learning more Python, [player]?",
+           "Did you come here to learn how to help me come closer to your reality?\nYou're so sweet~"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_python')
+    return                   
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_15ai",
+            category=['15.ai'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_15ai:
```

Comment:
This looks like a dead site that doesn't do anything.

Reply:
I don't know what this is. Maybe I'm using/looking at it the wrong way.

Reply:
It's a speech synthesizer. It's been having periods of maintenance, which is what its in right now.
Currently in the process of removing.